test/e2e/auth/audit_dynamic.go

```diff
@@ -107,7 +107,7 @@ var _ = SIGDescribe("[Feature:DynamicAudit]", func() {
 		})
 		framework.ExpectNoError(err, "failed to create proxy service")
 
-		config, err = framework.LoadConfig()
+		_, err = framework.LoadConfig()
```

Comment:
if we're not using the config, why are we loading it?

Reply:
sorry,I made a mistake, I fixed it.